### PR TITLE
Adds Maintenance lights

### PR DESCRIPTION
**Pull request by IamVelcroboy**
Changelog:

- Adds 2 variants of maintenance lights (red and white)
- Adds appropriate boxes/crates/cargo
- Tweaks black and red colored lights (reason is so you can have the same effect with less # of lights mapped.)
- Organizes all Nyano-specific lights into 1 location/file.
- Updates appropriate spawners

### CODE REVIEW

File: Resources/Prototypes/Nyanotrasen/Catalog/Fills/Boxes/general.yml [hunk 1]
```diff
@@ -24,4 +24,29 @@
   - type: Tag
     tags:
       - DroneUsable
+
+- type: entity
+  name: maintenance lightbulb box
+  parent: BoxCardboard
+  id: BoxMaintenancelightbulb
```
Comment (Elijahrane) [inline Resources/Prototypes/Nyanotrasen/Catalog/Fills/Boxes/general.yml]: inconsistent camelcasing here, you alternate between 'Maintenancelight... and MaintenanceLight...'. Please choose one (probably the second) and use it for all protos

**Review by Elijahrane — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Nyanotrasen/Catalog/Fills/Boxes/general.yml (hunk 1 above)
Comment (IamVelcroboy) [reply Resources/Prototypes/Nyanotrasen/Catalog/Fills/Boxes/general.yml]: That was an oops. I fixed all the other cases I could find as well.

File: Resources/Prototypes/Nyanotrasen/Markers/Spawners/Random/randomitems.yml [hunk 2]
```diff
@@ -186,4 +192,5 @@
       - ClothingHandsGlovesNitrile
       - ClothingOuterStraightjacket
       - ClothingShoesGaloshes
+      - WeaponDisablerPractice
```
Comment (Elijahrane) [inline Resources/Prototypes/Nyanotrasen/Markers/Spawners/Random/randomitems.yml]: out of scope and undocumented change?

**Review by Elijahrane — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Nyanotrasen/Markers/Spawners/Random/randomitems.yml (hunk 2 above)
Comment (IamVelcroboy) [reply Resources/Prototypes/Nyanotrasen/Markers/Spawners/Random/randomitems.yml]: Ooo, somehow that must have bled through from another branch where I was updating the spawners with new items. I was planning to add it anyway. Do you need me to remove?

Comment (Elijahrane) [reply Resources/Prototypes/Nyanotrasen/Markers/Spawners/Random/randomitems.yml]: remove it from this pr

Comment (IamVelcroboy) [reply Resources/Prototypes/Nyanotrasen/Markers/Spawners/Random/randomitems.yml]: Is done

**Review by Elijahrane — APPROVED** [review]

**Pull request merged**